lib/spawn-runner.js

```diff
@@ -49,10 +49,7 @@ const readFileAsync = Promise.promisify(readFile);
 const unlinkAsync = Promise.promisify(unlink);
 
 export function convertPathToPattern(path :string) :string {
-  if (process.platform.match(/^win/)) {
-    return path.replace(/\\/g, '\\\\');
-  }
-  return path;
+  return path.replace(/[|\\{}()[\]^$+*?.]/g, '\\$&');
```

Comment:
The pattern will work for windows, but not for unix. 
![screenshot_2017-05-29_20-02-17](https://cloud.githubusercontent.com/assets/4711633/26558536/d54d0b8c-44a9-11e7-86aa-c84018a72469.png)

Please return the if condition.